src/sys/signal.rs

```diff
@@ -619,6 +619,13 @@ impl SigSet {
             Signal::try_from(signum.assume_init()).unwrap()
         })
     }
+
+    /// Converts a sigset_t object to a SigSet without checking  whether the sigset_t is already
```

Comment:
Can you add links to the functions referenced in this documentation, or at least surround them with backticks so they render as code with `cargo doc`?

Thanks!